nnstreamer_example/custom_example_LSTM/meson.build

```diff
@@ -1,6 +1,6 @@
 library('dummyLSTM',
   'dummy_LSTM.c',
-  dependencies: [libm_dep],
+  dependencies: [libm_dep, nnstreamer_base_deps],
```

Comment:
do we need nnstreamer-dep for custom-filters?

Reply:
It is there accidentally. I will delete it.